privet/windows.go

```diff
@@ -9,25 +9,104 @@
 package privet
 
 import (
-	"errors"
+  "fmt"
+  "sync"
+  "github.com/andrewtj/dnssd"
```

Comment:
I am happy to go in either order. My thought was that if we moved Mac over first (which works today), we would find potential issues that might cause problems on Windows, thus improving the chances Windows would work properly.